### DISCUSSION TITLE
Jellyfin add option to disable hardware acceleration

### DISCUSSION THREAD

**Discussion by Forsskieken** [tteck/Proxmox#237] [Q&A]
Hi There,

I don't know if this is the place but I think Jellyfin fails because I can't do hardware acceleration on my Dell 520?
 The script fails after...
 ✓ Installed Dependencies
 - Setting Up Hardware Acceleration.../bin/chgrp: changing group of '/dev/dri': Operation not permitted
‼ ERROR 1@74 Unknown failure occured.

Is there a solution for this problem?

**Comment by tteck** [comment]
Try again 🤞🏻

**Comment by Forsskieken** [comment]
Hi There,

Thanks for the quick response, unfortunately it didn't work.

I've got the same error..
 ✓ Installed Dependencies
 - Setting Up Hardware Acceleration.../bin/chmod: changing permissions of
'/dev/dri': Operation not permitted
‼ ERROR ***@***.*** Unknown failure occured.
***@***.***:~#

The container is unprivilliged. Advanced or auto setup generates the same
error

Kind regards

**Reply by tteck** [reply]
It should be privileged for device pass-through
I take a look

**Reply by Forsskieken** [reply]
Hi, the latest changes broke the script..

bash: -c: line 141: syntax error near unexpected token `('

**Reply by tteck** [reply]
fixed

**Comment by Forsskieken** [comment]
Thanks for all the efforts, but unprivileged I still get error

**Reply by tteck** [reply]
It needs to be Privileged. Unprivileged will not see any devices.

**Comment by Forsskieken** [comment]
Okay thanks for clarifying that for me..
